package.json

```diff
@@ -21,7 +21,7 @@
     "start": "npm-run-all --parallel webpack:watch electron:serve",
     "webpack:watch": "webpack --watch",
     "start:web": "webpack-dev-server --content-base . --port 4200 --inline",
-    "build:electron:main": "tsc main.ts --outDir dist && copyfiles package.json dist && cd dist && npm install --prod && cd ..",
+    "build:electron:main": "tsc main.ts --outDir dist && copyfiles package.json dist && copyfiles -f src/app/providers/mock-data/boards.data.json dist/mock-data && cd dist && npm install --prod && cd ..",
```

Comment:
This will create a JSON data file in the build directory that is used by the app when it is running